src/tools.c

```diff
@@ -1350,7 +1375,11 @@ struct sockaddr_storage *str2sa_range(const char *str, int *port, int *low, int
 		memprintf(err, "dgram-type address not acceptable in '%s'", str);
 		goto out;
 	}
-
+// TODO: bypass si mptcp + use mptcpv4/6
+	if (mptcp){
+		printf(" avec mptcp \n");
+		// new_proto = &proto_mptcpv4;
+	}
```

Comment:
you don't need this, but instead, you can change the code below assigning `new_proto` with:

```c
		new_proto = protocol_lookup(ss.ss_family,
					    proto_type,
					    ctrl_type == SOCK_DGRAM || !!mptcp);
```